dockstore-common/src/main/java/io/dockstore/common/yaml/AbstractYamlService.java

```diff
@@ -69,6 +70,8 @@ public abstract class AbstractYamlService {
 
     private boolean latestTagAsDefault = false;
 
+    private Boolean disableDoiGeneration;
```

Comment:
What's the use case for `disableDoiGeneration`?  To be able to shut off the auto DOI generation if it's gone sideways, or something like that?